src/sections/reference.html

```diff
@@ -0,0 +1,41 @@
+<div class="relative">
+  <img
+    class="h-sm-reference md:h-md-reference lg:h-lg-reference"
+    src="/img/Rectangle-3.webp"
+    alt="a yellow rectangle"
+  />
+  <div class="flex justify-end items-end">
+    <ul
+      class="font-body text-right px-8 text-xl absolute top-0 pt-20 gap-3 w-w-46-reference md:w-w-reference md:right-24 md:bottom-28 md:text-2xl lg:bottom-36 lg:text-2xl lg:right-24">
+      <li><a href="#">Ace Hotel</a></li>
+      <li><a href="#">ARMEDANGELS</a></li>
+      <li><a href="#">BERLIN DESIGN WEEK</a></li>
+      <li><a href="#">Berliner Berg</a></li>
+      <li><a href="#">BITE CLUB</a></li>
+      <li><a href="#">Budweiser Budvar</a></li>
+      <li><a href="#">ChungKing Noodles</a></li>
+      <li><a href="#">Designpreis Brandenburg</a></li>
+      <li><a href="#">Die Techniker</a></li>
+      <li><a href="#">European Street Food Awards</a></li>
+      <li><a href="#">Grundmann Dentistry</a></li>
+      <li><a href="#">HORNBACH Werkstück</a></li>
+      <li><a href="#">HORNBACH macht Schule</a></li>
+      <li><a href="#">iF Design</a></li>
+      <li><a href="#">Kopka</a></li>
+      <li><a href="#">KLH Maßschuhe</a></li>
+      <li><a href="#">Lode & Stijn</a></li>
+      <li><a href="#">Luya</a></li>
+      <li><a href="#">MEISSEN</a></li>
+      <li><a href="#">Motel Beer & Coffee</a></li>
+      <li><a href="#">PAPER & TEA</a></li>
+      <li><a href="#">rocket & basil</a></li>
+      <li><a href="#">Seeberger Gruppe</a></li>
+      <li><a href="#">Shior</a></li>
+      <li><a href="#">SPOC Magazine</a></li>
+      <li><a href="#">stocubo</a></li>
+      <li><a href="#">STUR</a></li>
+      <li><a href="#">World of Coffee</a></li>
+      <li><a href="#">yamo</a></li>
```

Comment:
Since it's a big list, it's better if you can create a simple javascript file and map through them for best practice. 
N.B: this is a suggestion.